server/test/services/group_permissions.service.spec.ts

```diff
@@ -27,11 +27,7 @@ describe('GroupPermissionsService', () => {
     it('should validate uniqueness of group permission group name', async () => {
       const { adminUser } = await setupOrganization(nestApp);
 
-      const data = await service.create(adminUser, 'avengers');
-
-      expect(data.id).toBeDefined();
```

Comment:
Does this fail now?

Reply:
for update request we are not expecting any response params in front end

Reply:
so returning empty response